src/client/components/home/Meal.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+function Meal({ meal }) {
+  return (
```

Comment:
```suggestion
  const { id, title, description, price, location } = meal;
  return (
```

Using destructuring to unpack this might be a nice touch.